docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3.9'
+services:
+  ms-sql-server:
+    image: mcr.microsoft.com/mssql/server:2019-latest
+    environment:
+      ACCEPT_EULA: "Y"
+      SA_PASSWORD: "Password123!"
+      MSSQL_PID: Express
+    ports:
+      - 1433:1433
+    volumes:
+      - //c/Users/Илья/WordContext-03daec14-a505-48b7-8fb2-72bc85eb0fcd
```

Comment:
Can you rewrite definition of volume to not depend on your local path? Check this documentation https://docs.docker.com/compose/compose-file/compose-file-v3/#volumes

Reply:
Done.

Reply:
Done.